libs/ui-watt/src/lib/foundations/typography/+storybook/storybook-typography-overview.component.ts

```diff
@@ -14,121 +14,140 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Component } from '@angular/core';
+import { Component, ViewEncapsulation } from '@angular/core';
 
 import { typographyHtmlSnippets } from './shared/typography-html-snippets';
 
 interface Typography {
-  name: string;
   html: string;
   size: string;
   weight: string;
   letterCase: string;
   letterSpacing: string;
-  lineHeight: string;
 }
 
-const typeScale: Typography[] = [
+const typeScaleSmall: Typography[] = [
   {
-    name: 'Headline 1',
     html: typographyHtmlSnippets.h1.tag,
-    size: '36px',
-    // eslint-disable-next-line sonarjs/no-duplicate-string
-    weight: 'Semibold (600)',
-    letterCase: 'All caps',
+    size: 'XL',
+    weight: 'Semi-bold',
+    letterCase: 'Sentence',
     letterSpacing: '0',
-    lineHeight: '54px*',
   },
   {
-    name: 'Headline 2',
     html: typographyHtmlSnippets.h2.tag,
-    size: '32px',
-    weight: 'Semibold (600)',
-    letterCase: 'All caps',
+    size: 'L',
+    weight: 'Semi-bold',
+    letterCase: 'Sentence',
     letterSpacing: '0',
-    lineHeight: '48px*',
   },
```

Comment:
suggestion:nitpick: "headline-1" and "headline-2" on small screen have "bold" `font-weight` in Figma (though it still appears to be "semi-bold"). Align with Kenneth what should be the correct `font-weight`.

![image](https://user-images.githubusercontent.com/1096332/153608439-9091b834-bb3d-42ec-a3dd-47b3d95bede9.png)